Assets/MixedRealityToolkit.SDK/Inspectors/UX/Interactable/InteractableInspector.cs

```diff
@@ -89,16 +89,17 @@ public virtual void RenderCustomInspector()
             bool showStates = false;
             SerializedProperty states = serializedObject.FindProperty("States");
             bool drawerStarted = false;
+            string statesPrefKey = "Settings_States";
+            bool prefsShowStates = EditorPrefs.GetBool(statesPrefKey);
+
             if (states.objectReferenceValue != null)
             {
-                string statesPrefKey = "Settings_States";
-                bool prefsShowStates = EditorPrefs.GetBool(statesPrefKey);
                 EditorGUI.indentLevel = indentOnSectionStart + 1;
```

Comment:
Can you elaborate a bit more (on the PR, maybe not in comments in the code) about why an indent happens here but not down below?

Reply:
That's an oversight. I'll update it.